src/FormField/RegisteredMFAMethodListField.php

```diff
@@ -70,11 +70,12 @@ public function getSchemaDataDefaults()
     /**
      * Get the registered backup method (if any) from the currently logged in user.
      *
+     * @param Member $member
      * @return RegisteredMethod|null
      */
-    protected function getBackupMethod(): ?RegisteredMethod
+    protected function getBackupMethod(Member $member): ?RegisteredMethod
```

Comment:
We can't add properties to protected methods in a minor release, as that's a violation of our [public API](https://docs.silverstripe.org/en/project_governance/public_api/). This change would need to happen in a major release.

Is there a way you can fix this bug without modifying the existing API method signatures?

Reply:
@GuySartorelli I'm now storing Member in a private property and reverted the method signature back to the original. Let me know if that's ok.